nuxt/assets/css/main.css

```
@@ -0,0 +1,102 @@
+  @apply border-t border-hairline;
+}
+
+/*
+ * Content images come from Drupal at their original size, which is usually far
+ * larger than the column they land in. Cap the height as well as the width, so
+ * a tall image cannot push the rest of the page off the screen.
+ */
+.prose-content img,
+main img {
+  @apply max-w-full h-auto rounded;
+  max-height: 60vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)